driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java

```diff
@@ -86,10 +89,13 @@
 
   private AtomicInteger[] scheduledForCreation;
 
-  private final EventExecutor timeoutsExecutor;
+  private final EventExecutor scheduler;
```

Comment:
Please don't rename variables like that. We're rebasing the driver on top of Cassandra driver every now and then. Renames like this can cause conflicts and make it difficult to sync with Cassandra driver.